docs/demo/data/processed/Groundwater_h.py

```diff
@@ -0,0 +1,227 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Dec 24 10:09:12 2023
+
+@author: leyan
+"""
+
+from wsimod.nodes.storage import Storage
+from wsimod.core import constants
+
+class Groundwater_h(Storage):
```

Comment:
No idea but, given that this is a custom version of `Groundwater`, would it be helpful to inherit from `Groundwater` and not storage? That way we can somewhat consider that your custom `GroundwaterH` can be used wherever the built-in `Groundwater` is used, just doing things differently.